recipes/create-dmg/all/conanfile.py

```diff
@@ -13,34 +13,39 @@ class CreateDmgConan(ConanFile):
     topics = "command-line", "dmg"
     homepage = "https://github.com/create-dmg/create-dmg"
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os", "arch", "compiler", "build_type"
+    settings = "os"
     exports_sources = 'patches/**'
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def layout(self):
+        pass
```

Comment:
We always should have a layout for 2.0 :)

```suggestion
        basic_layout(self)
```